bin/nwjs_install

```diff
@@ -30,33 +30,43 @@ fi
 ARCH="x64"
 if [ `uname -m` == "x86_64" ]; then
 	ARCH="x64"
+elif [ `uname -m` == "aarch64" ]; then
+    ARCH="arm64"
 else
 	ARCH="ia32"
 fi
 
 # ready to go
 echo "Downloading nwjs..."
 set +e
-main_uri="https://dl.nwjs.io/$VERSION/nwjs-$VERSION-$OS-$ARCH.tar.gz"
-curl -sL -o $TEMPFILE $main_uri
-status=$?
-if [ $status -ne 0 ] ; then
-	echo "Failed to download '$main_uri': $status"
-	echo "Using mirror instead..."
-	mirror_uri="https://npm.taobao.org/mirrors/nwjs/$VERSION/nwjs-$VERSION-$OS-$ARCH.tar.gz"
-	curl -sL -o $TEMPFILE $mirror_uri
+if [ "$ARCH" == "arm64" ]; then
+	main_uri="https://github.com/LeonardLaszlo/nw.js-armv7-binaries/releases/download/nw60-arm64_2022-01-08/nw60-arm64_2022-01-08.tar.gz"
```

Comment:
hmm, yes. Problem is that the url for the arm version is a completely different one. So I decided to stick to a fixed version for arm but keep it as it was for the win, linux, osx releases.

Reply:
Yes, makes sense. That's why the `mv` command in line 47 doesn't use the variables.

Reply:
What do you think about an `ARM_VERSION` variable? And a wildcard (`*`) in line 47.

Reply:
I see your point. But I think that this will not help a lot because the download link also includes a date. Other Versions will have a completely different date in uri. Perhaps future releases of nwjs will also include linux-arm64 Versions. Then we can roll back and use the same logic for all versions.